### DISCUSSION TITLE
The switching actuators can only be switched once.

### DISCUSSION THREAD

**Discussion by zshermann** [SukramJ/aiohomematic#1207] [General]
### I agree to the following

- [X] I have read the [documentation](https://github.com/danielperna84/custom_homematic#custom_homematic)
- [X] I am aware of the latest [release notes](https://github.com/danielperna84/custom_homematic/releases)
- [X] The Backend (CCU/Homegear/...) is working as expected and connected devices are controllable by its own UI.
- [X] I am running the latest version of the custom_component (and Home Assistant)

### The problem

The switching actuators can only be switched once. After that, switching is no longer possible. The status of the switching actuators in the HA is also not updated when the actuator is changed via Raspmatic.

HA version:
Home Assistant 2023.9.3
Supervisor 2023.09.2
Operating System 10.5
Frontend 20230911.0 - latest

### What version of HomematicIP (local) has the issue?

1.42

### What type of installation are you running?

Home Assistant OS

### What type of installation are you running for your homematic backend?

RaspberryMatic Standalone

### Which version of your homematic backend are you running?

3.71.12.20230826 (RPI3)

### What hardware are you running for your system?

Pi3 Raspmatic, Pi4 HA

### Which config details do you use

- [X] Username and password
- [ ] TLS
- [ ] callback data ([see](https://github.com/danielperna84/custom_homematic#callback_host-and-callback_port))

### Which interfaces do you use?

- [X] Homematic IP
- [ ] Homematic (Bidcos-RF)
- [ ] Groups (Heating-Group)
- [ ] BidCos-Wired (HM-Wired)

### Diagnostics information

[home-assistant_homematicip_local_2023-09-25T16-45-56.201Z.log](https://github.com/danielperna84/hahomematic/files/12725007/home-assistant_homematicip_local_2023-09-25T16-45-56.201Z.log)

**Comment by SukramJ** [comment]
> Don't switch to DEBUG level if not requested.

Bitte keine Logs mit DEBUG Level anhängen! Ein einfaches HA log enthält alle Informationen.

Bitte die Integration Diagnostics anhängen (Klick auf die Integration ...)

Um welchen Aktor soll es sich eigentlich handeln? HmIP-XXX und SN.

Informationen die man im Log sieht:
```
2023-09-25 14:57:43.748 WARNING (MainThread) [hahomematic.central] PING/PONG MISMATCH: There is a mismatch between send ping events and received pong events for HA instance raspmatic. Possible reason 1: You are running multiple instances of HA with the same instance name configured for this integration. Re-add one instance! Otherwise one HA instance will not receive update events from your CCU. Possible reason 2: Something is stuck on CCU, so try a restart.
2023-09-25 15:24:23.841 WARNING (SyncWorker_3) [hahomematic.central] GET_LOCAL_IP: Can't resolve host for raspmatic
```
Bitte die zu diesem Problem dokumentierte [Beschreibung](https://github.com/danielperna84/custom_homematic#what-is-the-meaning-of-pingpong-mismatch-on-interface) durchlesen.

**Comment by SukramJ** [comment]
Bist Du sicher das das Problem noch existent ist?

Zumindest das Gerät 00351F2994F1AA empfängt seit 18:43 Uhr Änderungen.

**Comment by zshermann** [comment]
Ja. Das ist ja was ich nicht verstehe. Laut Protokoll funktioniert alles, aber in der Wirklichkeit: NÖ.

Also ich habe gestern Nachmittag mein HA KOMPLETT neu installiert. 
HA OS Image, HACS, HomematicIP neu.

Bei HM IP hatte ich die Option “aktiviere Sysvar Scan” und " aktiviere Systembenachrichtigungen" deaktiviert.  Da ging alles (18:43)

Zufrieden ins Bett gegangen. :)

Heute früh habe ich die Optionen wieder aktiviert und es geht wieder nix. :(
Darum habe ich jetzt den Issue aufgemacht.

**Comment by SukramJ** [comment]
Nochmal: Um welche(s) Gerät(e) geht es?

Die Optionen haben nichts mit deinem Problem zu tun, ausser das die Integration neu geladen wird.
Das Problem ist, das HA keine Events von der CCU/RM empfängt.

Gibt es mehrere Instanzen von HA die auf die CCU zugreifen?  Ggf. mal die CCU neustarten.

**Comment by SukramJ** [comment]
Und häng bitte die Integration Diagnostics an.

**Comment by Baxxy13** [comment]
`2023-09-25 15:24:23.841 WARNING (SyncWorker_3) [hahomematic.central] GET_LOCAL_IP: Can't resolve host for raspmatic`

Klingt ja irgendwie nach Netzwerk / DNS-Problemen.
Was ist denn bei der Integration eingetragen?
Hostname oder IP-Adresse?

**Comment by zshermann** [comment]
@Baxxy13:
Netzwerk würde ich ausschließen. Ich gehe über IP. Beide RPi sind im gleichen Netz und auf der gleichen Switch angeschlossen.
Pingzeiten bei 0,5 - 0.9 ms :)

**Comment by SukramJ** [comment]
Da ich nicht von einem Fehler in der Integration ausgehe, werde ich dieses Issue in eine Diskussion umwandeln.

**Comment by zshermann** [comment]
@SukramJ : Meinst Du die ?

[config_entry-homematicip_local-b22e5be953da04ca7a5a8dab9c2c360c.json.txt](https://github.com/danielperna84/hahomematic/files/12727488/config_entry-homematicip_local-b22e5be953da04ca7a5a8dab9c2c360c.json.txt)

**Reply by Baxxy13** [reply]
Interessanter IP-Adress-Bereich.
Ich würde mal die Werte folgender Einträge im Config-Flow entfernen.
```json
"callback_host": "172.30.30.110",
"callback_port": 2010,
```
Wenn man nicht weiß was es damit auf sich hat ist es besser das einfach leer zu lassen.

Vermutlich schickt die Zentrale nach der ersten Kontaktaufnahme ihre Events nur noch an sich selbst. 😉 
Dabei wird sie die Events nicht los, füllt ihre Buffer und kommt vermutlich irgendwann zum erliegen.

**Reply by SukramJ** [reply]
> @SukramJ : Meinst Du die ?

JA

Wie @Baxxy13 schon sagte, beide Felder leer lassen. Die Werte machen absolut keinen Sinn.

**Reply by Baxxy13** [reply]
Dieser "Event-Loop" dürfte auch das 1x funktionieren erklären.

**Comment by zshermann** [comment]
callback... entfernt und jetzt geht es.
